### ISSUE TITLE
Review loading namespaces

### ISSUE THREAD

**Issue by aralroca** [vinissimus/next-translate#168]
https://github.com/vinissimus/next-translate/issues/167#issuecomment-631358050

**Comment by BjoernRave** [comment]
After taking a closer look to the output I realized, that it's only loading additional namespaces for index routes, so it's not related to regex. This is my config:

```
{
  "allLanguages": ["en", "pt", "es"],
  "defaultLanguage": "es",
  "currentPagesDir": "pages_",
  "finalPagesDir": "pages",
  "localesPath": "public/locales",
  "defaultLangRedirect": "lang-path",
  "logBuild": false,
  "pages": {
    "*": ["common"],
    "/": ["forms", "account"],
    "/dashboard": ["analytics", "forms"],
    "/account/settings": ["forms", "account"],
    "/account/globalsettings": ["forms", "table", "account"],
    "/logout": ["forms", "account"],
    "rgx:/new$": ["forms", "table"],
    "rgx:/new-multi$": ["forms", "table"],
    "rgx:/import$": ["forms", "table"],
    "rgx:/\\[id\\]/edit$": ["forms", "table"],
    "/role/[id]": ["forms", "table"],
    "rgx:/\\[id\\]$": ["table", "analytics"],
    "rgx:/\\[id\\]/dashboard$": ["analytics"],
    "/inventory": ["table"],
    "/movement": ["table", "forms"],
    "/client": ["table"],
    "/supplier": ["table"],
    "/product": ["table"],
    "/transportagency": ["table"],
    "/user": ["table"],
    "/warehouse": ["table"],
    "/role": ["table"],
    "/invoicepdf": ["table"],
    "/import": ["table"],
    "/invoices": ["table"]
  }
}
```
Even routes like `/dashboard` where it's explicitly only for this route only receive the common namespace

**Comment by aralroca** [comment]
@BjoernRave it should be fixed on `0.16.0-canary.7`. Would be great if you can confirm 👏

It was introduced by a typo in a regex on https://github.com/vinissimus/next-translate/pull/163 . I added the failing test to cover this in the future.

**Comment by BjoernRave** [comment]
@aralroca great new, I will give it a try :)
